src/D3MHub.sol

```diff
@@ -0,0 +1,604 @@
+// SPDX-FileCopyrightText: © 2021-2022 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021-2022 Dai Foundation
```

Comment:
Consider keeping file creation year (for aave it should be 2021) and removing old copyright comment (as per Dai Foundation copyright guidance), in source files including tests.
```suggestion
// SPDX-FileCopyrightText: © 2021 Dai Foundation <www.daifoundation.org>
// SPDX-License-Identifier: AGPL-3.0-or-later
```

Reply:
Addressed in #75